tests/l2/test_l2_configure.py

```diff
@@ -0,0 +1,137 @@
+"""
+Tests related to L2 configuration
+"""
+
+import logging
+import pytest
+
+from tests.common import config_reload
+from tests.common.platform.processes_utils import wait_critical_processes
+from tests.common.helpers.assertions import pytest_assert
+
+CONFIG_DB = "/etc/sonic/config_db.json"
+CONFIG_DB_BAK = "/etc/sonic/config_db.json.bak"
+DUT_IMG_PATH = "/tmp/dut-sonic-img.bin"
+LOCALHOST_IMG_PATH = "/tmp/localhost-sonic-img.bin"
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology("any"),
```

Comment:
>any [](http://example.com/codeflow?start=26&length=3)

we can limit to t0 topo. no need to test l2 mode on t1 topo.

Reply:
Thanks! Done.